src/search.rs

```diff
@@ -21,7 +21,7 @@ pub fn top_down<'d, Tgt, D>(
     db: &'d D,
     goal: &Spec<Tgt>,
     top_k: usize,
-    parallel: bool,
+    jobs: Option<usize>,
 ) -> (SmallVec<[(ActionIdx, Cost); 1]>, u64, u64)
```

Comment:
Is behavior if `Some(0)` is given defined? If not, should we use `Option<NonZeroU32>`?

Reply:
Oh, I missed this case. Thank you!

Do you mind if I use `NonZeroUsize` instead of `NonZeroU32` since `rayon::current_num_threads` returns `usize`?

Reply:
sgtm.